Separating slow tests from faster running tests

RabeaGleissner opened the pull request:
@felipesere @jsuchy

I moved the two slower running tests to a separate file and am using the Maven Surefire plugin to exclude that file from the default test run. To run all tests, the command `mvn integration-test` needs to be used.

RabeaGleissner committed:
moves slower tests in separate file and specifies integration test phase in pom

RabeaGleissner committed:
adjust tests to reflect intention

RabeaGleissner committed:
updates instructions on how to run the tests

RabeaGleissner committed:
delete unused imports

jsuchy commented on src/test/java/de/rabea/player/ComputerPlayerSpeedTest.java:
```diff
@@ -0,0 +1,38 @@
+        }
+        long endTime = System.nanoTime();
+        long duration = (endTime - startTime)/1000000;
+        System.out.println("duration in milliseconds= " + duration);
```
Not sure about leaving these println statements in these tests. At the very least, when you run all the slow tests, there's no context about what each of the durations are for.

RabeaGleissner replied:
@jsuchy took them out :+1:

RabeaGleissner committed:
removes print statements

felipesere commented:
I would take this one step further:
Create a function that takes a board an measures how long it takes, then use it with `new Board(3)` and `new Board(4)` respectively. The duplication between those two tests is not needed.

RabeaGleissner committed:
removes duplication from speed tests

RabeaGleissner commented:
@felipesere Thanks for the feedback. I've removed the duplication. I also changed the test for 3x3 to only run the `computer.getPosition()` method once.

felipesere commented:
Its looking good.
@jsuchy question about taste: not having an `assert` call looked weird to me. But now that I think of it, the assert will never 'fail' as the the `@Timeout(...)` will trigger first. 
Would you leave the assertion in anyway?

jsuchy commented:
I would kill assertions (similar to a test that ensures an exception is thrown, the "assertion" is in the annotation).

RabeaGleissner committed:
removes assertions and helper method
